tests/virt_autotest/libvirt_isolated_virtual_network.pm

```diff
@@ -27,14 +27,14 @@ sub run_test {
     virt_autotest::virtual_network_utils::download_network_cfg($vnet_isolated_cfg_name);
 
     #Stop named.service, refer to poo#175287
-    systemctl("stop named.service") if (is_sle('15+'));
+    systemctl("stop named.service") if (is_sle('>=15-SP6'));
```

Comment:
Thank you @guoxuguang for the workaround enhancement. Would you please have a look at https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/21269?  Such named operation fails MU jobs like [this](https://openqa.suse.de/tests/16836568#step/libvirt_nated_virtual_network/17). Can you consider some fix for this?

Reply:
Cool, thanks

I also triggered a test with your PR to the test, let's see how it goes.

```
qa2-fix-5:~ # openqa-clone-custom-git-refspec https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/21288 https://openqa.suse.de/tests/16836568 -c '--host openqa.suse.de' BUILD=":222222:none" WORKER_CLASS=live_migrat_2
1 job has been created:
 - sle-15-SP6-Server-DVD-Incidents-VIRT-Core-x86_64-Build:666666:none-qam-xen-install-and-features-test@64bit-ipmi-legacy-boot -> https://openqa.suse.de/tests/16946467
qa2-fix-5:~ # 

```

Reply:
The job passed.